WeatherApp/WeatherApp/Feature/CityScreen/CityScreenView.swift

```diff
@@ -0,0 +1,86 @@
+import SwiftUI
+
+struct CityScreenView: View {
+
+    @ObservedObject var viewModel: CityScreenViewModel
+
+    let columns = [
+        GridItem(.flexible(), spacing: 18),
+        GridItem(.flexible())]
+
+    var body: some View {
+        ZStack {
+            Color.black
+                .ignoresSafeArea()
+
+            if let weather = viewModel.weather {
+                ScrollView {
+                    VStack {
+
+                        NavBar(backAction: viewModel.goBack)
+
+                        Text(viewModel.city)
+                            .font(Font.custom("NDOT45inspiredbyNOTHING", size: 25))
+
+                        Image(.sunny)
+                            .resizable()
+                            .scaledToFit()
+                            .frame(maxWidth: 170, maxHeight: 170)
+                            .padding()
+
+                        Text(weather.description.uppercased())
+                            .font(Font.custom("Noto Sans Mono", size: 12))
+                            .padding(.bottom, 20)
```

Comment:
Wrap this in a VStack and extract it to its own computes property

```
private var mainInfo: some View {
    VStack { ...
}
```

Also while extracting these values I would like you to pay attention where you could benefit from VStacks spacing in favor of paddings. Also keep those paddings in mind when extracting these components